fix: issue of broken images links UA-238

tusharkgeospoc opened the pull request:
Attempting to fix this broken images links issue using target as static in nuxt.config.js as described in following links.

- https://nuxtjs.org/docs/2.x/deployment/github-pages

Commit:
fix: issue of broken images links UA-238 #time 1h 10m

vinayakkulkarni reviewed (commented):
This won't solve the issue of path to the links

tusharkgeospoc commented:
> This won't solve the issue of path to the links

I have removed `target: static` now from `nuxt.config.js` file
I have converted and optimized `png` images into `svg` images
Its rendering as we need on my local system.
Would it work on GH pages of this repo if we merge this PR ??

vinayakkulkarni commented on content/en/1.index.md:
```diff
@@ -30,14 +30,14 @@ bulletLinks:
           <img
             class="object-contain"
-            src="/logos/geoqi.png"
+            src="/logos/geoqi.svg"
```
```suggestion
            :src="$config.env === 'production' ? '/gqi-mapper-documentation/logos/geoqi.png' : '/logos/geoqi.png'"
```

Commit:
feat: use `nuxt-img` to render the image
